custom-elements-manifest.config.js

```diff
@@ -28,8 +28,5 @@ export default {
         'node_modules/*',
         '**/*.dev.*',
     ],
-    outdir: '.',
-    litelement: true,
-    packagejson: false,
```

Comment:
can you share why these options were removed from the config? or share some docs for me to learn?